docs/integrations/mirroring.md

```diff
@@ -193,17 +194,17 @@ def get_mapping_fields_command():
 
 ## Incident fields on a XSOAR incident 
 The following incident fields must be configured either in the integration or the instance mapping:
-* dbotMirrorDirection - valid values are Both, In, or Out.
-* dbotMirrorId - represents the id of the incident in the external system.
-* dbotMirrorInstance - the instance through which you will be mirroring.
+* **dbotMirrorDirection** - valid values are Both, In, or Out.
+* **dbotMirrorId** - represents the id of the incident in the external system.
+* **dbotMirrorInstance** - the instance through which you will be mirroring.
 
 Useful fields:
-* dbotMirrorTags - tags for mirrored-out entries (comment/files).
+* **dbotMirrorTags** - tags for mirrored-out entries (comment/files).
 
 ## Debugging
 
-* getMirrorStatistics command - a hidden command that returns mirroring statistics: total mirrored, rate limited, and last run.
-* getSyncMirrorRecords - a hidden command that returns records that hold internal mirroring metadata for each mirrored incident
-* get-remote-data - is runnable through the war room if the command is defined in the yml file.
-* get-modified-remote-data - is runnable through the war room if the command is defined in the yml file.
-* get-mapping-fields - is runnable through the war room with no arguments and see the results.
+* **getMirrorStatistics** command - a hidden command that returns mirroring statistics: total mirrored, rate limited, and last run.
+* **getSyncMirrorRecords** - a hidden command that returns records that hold internal mirroring metadata for each mirrored incident
```

Comment:
Another hidden command we added now is `purgeClosedSyncMirrorRecords` which receives 2 arguments: mandatory `dbotMirrorInstance` for which integration to purge and optional `olderThan` in days. Since this bucket is forever growing (with all the closed incidents that are mirrored), might be a good idea to purge when you're sure that there will be no update from the remote system on the really old incidents.